src/facade/dragonfly_connection.cc

```diff
@@ -424,6 +424,12 @@ uint32_t Connection::GetClientId() const {
   return id_;
 }
 
+bool Connection::IsAdmin() const {
```

Comment:
can you please use this function inside `CheckForHttpProto` function?

Reply:
Good point!